rootfs/scheduler/k8s.py

```diff
@@ -301,46 +369,52 @@ def scale(self, name, image, command, **kwargs):
             self._scale_app(name, old_replicas, app_name)
             raise RuntimeError('{} (Scale): {}'.format(name, e))
 
-    def _create_rc(self, name, image, command, **kwargs):
-        container_fullname = name
-        app_name = kwargs.get('aname', {})
-        app_type = name.split('.')[1]
-        container_name = app_name + '-' + app_type
-        name = name.replace('.', '-').replace('_', '-')
-        args = command.split()
-
-        # First ensure that the namespace was created
-        url = self._api("/namespaces/{}", app_name)
-        resp = self.session.get(url)
-        if resp.status_code == 404:
+    def create_namespace(self, code, app_name):
+        if code == 404:
```

Comment:
The `404` would be better served outside this function (make the caller do the checking) and have this serve as a `create` function only.

Also note the convention in rest of the file is prefix the functions with `_` so we should keep with that